src/main/steps/applicant1/hub-page/joint/form.njk

```diff
@@ -23,14 +23,5 @@
       name: applicantApplyForConditionalOrderStarted,
       value: "Yes"
     }) }}
-  {% elif displayState.state() == 'AwaitingFinalOrder' %}
```

Comment:
how comes youre removing the button for the hub page?

Reply:
So am using the following link for the finalising your application buttonLink: `${isApplicant2 ? `${APPLICANT_2}${FINALISING_YOUR_APPLICATION}` : FINALISING_YOUR_APPLICATION}`, which is in the applicant1/hub-page/joint/content.ts. I followed what was written for the sole hub page.

Reply:
not sure what you mean by this, can you link whats written in the sole hub page?

Reply:
oh, i think i get what youre saying and that fine, but doesnt that mean you need to replace the button with something? I dont see any of that here.

Reply:
I reverted the change and fixed the button.